resources/metadata/maven.yaml

```diff
@@ -60,9 +60,3 @@ spec:
           - PARAMETERS
         default: false
         mandatory: false
-      - name: returnStdout
```

Comment:
Well, that was not what I meant by that. I guess we still have to keep that flag returnStdout for the step if we do not always return stdout. I guess users whould sill like to use the go step to evaluate maven projects

Reply:
ok. I'll revert the changes